src/pushsource/_impl/model/ami.py

```diff
@@ -13,6 +13,23 @@ class AmiRelease(VMIRelease):
     for backwards compatibility.
     """
 
+    @classmethod
+    def from_data(cls, data):
```

Comment:
Like it is with erratum.py, can we please name these `_from_data` so they're considered private?  We don't want other codebases to start relying on the dict structure.